deploy/playbooks/roles/common/tasks/main.yml

```diff
@@ -1,52 +1,41 @@
 ---
-- name: "Build hosts file"
-  sudo: yes
-  template:
-    src: hosts.j2
-    dest: "/etc/hosts"
-
-- name: Set Hostname with hostname command
```

Comment:
@djgalloway I'm not sure, maybe it won't be as much an issue for mita. I know it caused us some headache with chacra nodes for sure. As long as we can deploy and restart ``rabbitmq`` successfully without doing a hostname dance I'm ok with this.